zulip/integrations/bridge_with_slack/run-slack-bridge

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import sys
+import time
+import os
+import argparse
+import traceback
+import multiprocessing as mp
+import zulip
+from slackclient import SlackClient
```

Comment:
This should possibly be added as a requirement, or an import failure should print the exact version of `slackclient` to install. The latest version of it (2.x) seems to have changed the API, and this script needs `1.3.1`.

Reply:
The local requirements.txt already has the package, while adding this to global python-zulip-api's would be overkill. I am about to push the 2.x update, although not yet tested.

Reply:
Looks like you forgot to commit the local `requirements.txt`?

Reply:
`*facepalm*`